common/AddonMessage.ts

```diff
@@ -17,7 +17,7 @@ interface AddonMessage {
     findMatches?: { uri: string };
     findMatchesResult?: any;
     logins?: keeLoginInfo[];
-    action?: "primary" | "detectForms" | "manualFill" | "generatePassword" | "closeAllPanels" | "showMatchedLoginsPanel" | "removeSubmittedData" | "saveLatestLogin";
+    action?: Action.Primary | Action.DetectForms | Action.ManualFill | Action.GeneratePassword | Action.CloseAllPanels | Action.ShowMatchedLoginsPanel | Action.RemoveSubmittedData | Action.SaveLatestLogin;
```

Comment:
Could this simply be `action?: Action;`?

Reply:
I agree with @dlech - that would be better.